### PR TITLE
build: prepare v0.17.3.2

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

**Review by SChernykh — APPROVED** [review]

**Review by jeffro256 — APPROVED** [review]
Validated `checkpoints.dat` and `checkpoints.cpp`

File: src/checkpoints/checkpoints.cpp [hunk 1]
```diff
@@ -249,6 +249,7 @@ namespace cryptonote
     ADD_CHECKPOINT2(2325000, "f61261994b368700f0cbbfb4477433fa36a3c7537908ab2d1a06ac2987cc8b01", "0x154bceeffaff847");
     ADD_CHECKPOINT2(2430000, "5f7f4273eb6be024df3c4311a8d28482220a253190efa1719f35b7782587740c", "0x1c3f14ba9b0bebc");
     ADD_CHECKPOINT2(2478000, "692fc889f5328c9cfe47e9846ca2bb3d61d82dbeb37077e90e0f8c36b8fc0c84", "0x1fbb38d0aae41a6");
+    ADD_CHECKPOINT2(2606000, "41a212fe33f9317a702a163e4c45b6295a9cc303745fe21bfe499a5e39b119c7", "0x29e2348ffb1e034");
```
Comment (mj-xmr) [inline src/checkpoints/checkpoints.cpp:252]: Got the same hash using:
`curl http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"get_block_header_by_height","params":{"height":2606000}}' -H 'Content-Type: application/json'`

File: src/cryptonote_core/blockchain.cpp [hunk 2]
```diff
@@ -5432,7 +5432,7 @@ void Blockchain::cancel()
 }
 
 #if defined(PER_BLOCK_CHECKPOINT)
-static const char expected_block_hashes_hash[] = "dbc25cc509fe8a44426f7168c608d0b4cf6ea5d43682ba4087d6a47aa0384656";
+static const char expected_block_hashes_hash[] = "a6290211ca573e73c626f3fdeaed058b909bf57db5d9c44c1037e9190891b426";
```
Comment (mj-xmr) [inline src/cryptonote_core/blockchain.cpp:5435]: Got the same hash using:

```
./monero-blockchain-export --output-file /tmp/checkpoints.dat --block-stop 2606000 --blocksdat
sha256sum /tmp/checkpoints.dat 
```

**Review by mj-xmr — APPROVED** [review]
Double checked and documented the review process.

File: src/checkpoints/checkpoints.cpp (hunk 1 above)
Comment (SChernykh) [reply src/checkpoints/checkpoints.cpp:252]: I just realized it's even easier to check with https://xmrchain.net/block/41a212fe33f9317a702a163e4c45b6295a9cc303745fe21bfe499a5e39b119c7
```
Block hash (height): 41a212fe33f9317a702a163e4c45b6295a9cc303745fe21bfe499a5e39b119c7 (2606000)
```

Comment (mj-xmr) [reply src/checkpoints/checkpoints.cpp:252]: Too easy :)

**Pull request merged**